src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.html

```diff
@@ -1,164 +1,240 @@
+          </li>
+        </ul>
+      </ng-template>
+      <div class="health-popover-wrapper">
+        <div [ngStyle]="contentData.health.status | healthColor"
+              [popover]="contentData.health.checks.length > 0 ? healthChecks : ''"
+              triggers=""
+              #healthChecksTarget="bs-popover"
+              placement="bottom">
+          {{ contentData.health.status }}
+        </div>
       </div>
-    </div>
-  </div>
+    </cd-info-card>
 
-  <div class="row">
-    <cd-info-card title="Usage"
-                  *ngIf="contentData.df?.stats?.total_objects">
-      <table class="ceph-chartbox">
-        <tr>
-          <td>
-            <span style="font-size: 45px;">{{ contentData.df.stats.total_objects | dimless }}</span>
-          </td>
-          <td>
-            <div class="center-block pie">
-              <cd-health-pie [data]="contentData"
-                             (prepareFn)="prepareRawUsage($event[0], $event[1])"></cd-health-pie>
-            </div>
-          </td>
-          <td>
-            <div class="center-block pie">
-              <cd-health-pie [data]="contentData"
-                             (prepareFn)="preparePoolUsage($event[0], $event[1])"></cd-health-pie>
-            </div>
-          </td>
-        </tr>
-        <tr>
-          <td i18n>Objects</td>
-          <td>
-            <ng-container i18n>Raw capacity</ng-container>
-            <br>
-            <ng-container i18n="disk used">({{ contentData.df.stats.total_used_bytes | dimlessBinary }} used)</ng-container>
-          </td>
-          <td i18n>Usage by pool</td>
-        </tr>
-      </table>
+    <cd-info-card cardTitle="Monitors"
+                  link="/monitor"
+                  class="col-sm-6 col-md-4 col-lg-3"
+                  contentClass="content-highlight"
+                  *ngIf="contentData.mon_status">
+      {{ contentData.mon_status | monSummary }}
+    </cd-info-card>
+
+    <cd-info-card cardTitle="OSDs"
+                  link="/osd"
+                  class="col-sm-6 col-md-4 col-lg-3"
+                  contentClass="content-highlight"
+                  *ngIf="contentData.osd_map">
+      <span *ngFor="let result of (contentData.osd_map | osdSummary)" [ngStyle]="result.style">
+        {{ result.content }}
+      </span>
+    </cd-info-card>
+
+    <cd-info-card cardTitle="Manager Daemons"
+                  class="col-sm-6 col-md-4 col-lg-3"
+                  contentClass="content-highlight"
+                  *ngIf="contentData.mgr_map">
+      {{ contentData.mgr_map | mgrSummary }}
+    </cd-info-card>
+
+    <cd-info-card cardTitle="Hosts"
+                  link="/hosts"
+                  class="col-sm-6 col-md-4 col-lg-3"
+                  contentClass="content-medium content-highlight">
+      {{ contentData.hosts }}
+    </cd-info-card>
+
+    <cd-info-card cardTitle="Object Gateways"
```

Comment:
i18n-cardTitle is missing to support translating. Please check the other code for this behaviour, too.

Reply:
Done.